ci/cron/src/Main.hs

```diff
@@ -373,9 +382,14 @@ parser = info "This program is meant to be run by CI cron. You probably don't ha
                      (Check <$> Opt.strOption (Opt.long "bash-lib"
                                          <> Opt.metavar "PATH"
                                          <> Opt.help "Path to Bash library file.")
-                            <*> Opt.strOption (Opt.long "gcp-creds"
+                            <*> (Opt.optional $
+                                  Opt.option Opt.auto (Opt.long "gcp-creds"
```

Comment:
Why did you replace `strOption` by `option auto`?

Reply:
thinking about this, is this even equivalent? `auto` is based on `Read` but `Read` for `String` excepts a string literal including the quotes around it.

Reply:
You're right, it fails. There was no good reason to replace, either. At some point I hoped `option auto` would automatically handle the `Maybe`, but then I discovered `optional` and forgot to change back.